asmcli/lib/installation_dependencies.sh

```diff
@@ -265,7 +265,7 @@ generate_secret_name() {
 }
 
 register_cluster() {
-  if is_cluster_registered; then return; fi
+  if retry 2 is_cluster_registered; then return; fi
```

Comment:
Let's not retry entire functions unless it's absolutely necessary.

Reply:
Done